packages/expo-dev-menu/ios/DevMenuAppInstance.swift

```diff
@@ -76,9 +76,9 @@ class DevMenuAppInstance: DevMenuRCTAppDelegate {
     }
     // Return `nil` if the content is not a valid URL.
     guard let content = try? String(contentsOfFile: packagerHostPath, encoding: String.Encoding.utf8).trimmingCharacters(in: CharacterSet.newlines),
-      let url = URL(string: content) else {
+      let url = URL(string: "http://\(content)") else {
       return nil
     }
-    return url.absoluteString
+    return "\(url.host ?? ""):\(url.port ?? 8081)"
```

Comment:
`packages/expo-dev-menu/scripts/start.sh` generates a `dev-menu-packager-host` file in the following format `ip:port` e.g. `192.168.0.9:2137`, which is not a valid `URL(string)` in Swift, because of that we need to manually specify the scheme